tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -1359,6 +1389,13 @@ namespace vz_example_viewer {
      * json.
      */
     createCompareExamplesFromJson: function(json: string) {
+      // If setting the compare example back to empty, and there is saliency
+      // information, then set the compare info to the saliency so that it
+      // is displayed instead of nothing.
+      if (!json || (!Object.keys(json).length && this.saliencyJson)) {
```

Comment:
I think these two if statements could be simplified to make them easier to read.
(e.g. if !json then you always set json = this.saliencyJson which will enter the if below if !this.saliencyJson is true. )

Reply:
simplified